### ISSUE TITLE
entc generate template error

### ISSUE THREAD

**Issue by jordan0420** [facebookincubator/ent#445]
working through the Quick Introduction on https://entgo.io/docs/getting-started/
* User with no fields defined generates correctly.
* adding the 2 fields to the User schema step fails to generate:

go version info:
```go version go1.13.8 linux/amd64```

project/ent/schema/user.go

```go
package schema

import (
	"github.com/facebookincubator/ent"
	"github.com/facebookincubator/ent/schema/field"
)

// User holds the schema definition for the User entity.
type User struct {
	ent.Schema
}

// Fields of the User.
func (User) Fields() []ent.Field {
	return []ent.Field{
		field.Int("age").
			Positive(),
		field.String("name").
			Default("unknown"),
	}
}

// Edges of the User.
func (User) Edges() []ent.Edge {
	return nil
}

```

```bash
$ entc generate ./ent/schema
entc/gen: execute template "create": template: templates:39:47: executing "create" at <ne $f $.ID>: error calling ne: invalid type for comparison

```

**Comment by a8m** [comment]
Hey @jordan0420, 
Any reason for not using Go 1.14? Go 1.13 templates don't support comparing pointers.

**Comment by a8m** [comment]
Anyway, I've created #446 to fix this, but I'll revert it once Go 1.15 is out.

Thanks for reporting this, closing

**Comment by jordan0420** [comment]
Thanks for the quick resolution!
